### ISSUE TITLE
Cannot create PR request on github.dev

### ISSUE THREAD

**Issue by lychichem** [microsoft/vscode#149346]
ADD ISSUE DESCRIPTION HERE
Probably due to `Access to XMLHttpRequest at 'https://api.github.com/repos/l0o0/jasminum/branches?per_page=100' from origin 'https://v--19e0pdf0sf4kl586csmr0151v0pao86in6ve81ep1ffqki8v3p35.vscode-cdn.net' has been blocked by CORS policy: Request header field accept is not allowed by Access-Control-Allow-Headers in preflight response.` I cannot get branch/repo info and cannot create PR request by using vscode web on github.dev.
Version: 1.67.1
Commit: da15b6fd3ef856477bf6f4fb29ba1b7af717770d
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.4951.54 Safari/537.36 Edg/101.0.1210.39
Embedder: github.dev

**Comment by alexr00** [comment]
@lychichem can you please share a screenshot of what you're seeing?

**Comment by lychichem** [comment]
> @lychichem can you please share a screenshot of what you're seeing?

![image](https://user-images.githubusercontent.com/9316590/168279996-c94218de-6d8c-4ac6-861d-caf7427b1daa.png)
![image](https://user-images.githubusercontent.com/9316590/168280115-93029309-649b-4511-b190-dad12753c7ef.png)
test repo: https://github.com/lychichem/winget-pkgs/tree/test

**Comment by alexr00** [comment]
Duplicate of https://github.com/microsoft/vscode-pull-request-github/issues/3528.
